tensorflow_datasets/image/div2k_test.py

```diff
@@ -0,0 +1,161 @@
+"""Test for div2k dataset."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import div2k
+
+class Div2kTest_bicubic_x2(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = div2k.Div2k
+  BUILDER_CONFIG_NAMES_TO_TEST = ["bicubic_x2"]
+  SPLITS = {
+      "train": 1,
+      "validation": 1,
+  }
+
+  DL_EXTRACT_RESULT = {
+    "train_hr_url": "DIV2K_train_HR",
+    "valid_hr_url": "DIV2K_valid_HR",
+    "train_lr_url": "DIV2K_train_LR_bicubic_X2",
+    "valid_lr_url": "DIV2K_valid_LR_bicubic_X2",
+  }
+
+class Div2kTest_bicubic_x3(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = div2k.Div2k
```

Comment:
This testing is a little bit excessive ;-)

You don't need to test all the classes, as their code flow is the same in _generate_samples

Reply:
Should I get rid of all of the classes besides one?

Reply:
Yes, you can leave 1 or 2.
But we don't need so many.